dev/integration_tests/external_ui/test_driver/main_test.dart

```diff
@@ -0,0 +1,65 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'package:flutter_driver/flutter_driver.dart';
+import 'package:test/test.dart';
+
+final RegExp calibrationRegExp = new RegExp('Flutter frame rate is (.*)fps');
+final RegExp statsRegExp = new RegExp('Produced: (.*)fps\nConsumed: (.*)fps\nWidget builds: (.*)');
+const Duration samplingTime = const Duration(seconds: 3);
+
+Future<Null> main() async {
+  group('texture suite', () {
+    FlutterDriver driver;
+
+    setUpAll(() async {
+      driver = await FlutterDriver.connect();
+      // TODO(mravn): the following pause appears necessary on iOS to avoid
+      // inflating elements too early (when there is no size).
+      await new Future<Null>.delayed(const Duration(seconds: 1));
```

Comment:
this is going to cause a race and a flaky test. Can we just delay until we have a size?

Reply:
Done.